src/test/java/org/broadinstitute/hellbender/tools/walkers/readorientation/CollectDataForReadOrientationFilterIntegrationTest.java

```diff
@@ -0,0 +1,175 @@
+package org.broadinstitute.hellbender.tools.walkers.readorientation;
+
+import htsjdk.samtools.SAMFileHeader;
+import htsjdk.samtools.SAMReadGroupRecord;
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.utils.MathUtils;
+import org.broadinstitute.hellbender.utils.genotyper.*;
+import org.broadinstitute.hellbender.utils.read.ArtificialReadUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.read.ReadUtils;
+import org.broadinstitute.hellbender.utils.read.SAMFileGATKReadWriter;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.internal.junit.ArrayAsserts;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+
+/**
+ * Created by tsato on 8/1/17.
+ */
+public class CollectDataForReadOrientationFilterIntegrationTest extends CommandLineProgramTest {
+
+    /**
+     * Test the tool on a real bam to make sure that it does not crash
+     */
+    @Test
+    public void testOnRealBam() {
+        final File refTable = createTempFile("ref", ".table");
+        final File altTable = createTempFile("alt", ".table");
+
+        final String[] args = {
+                "-R", v37_chr17_1Mb_Reference,
+                "-I", NA12878_chr17_1k_BAM,
+                "-alt_table", altTable.getAbsolutePath(),
```

Comment:
You should extract public static constants for these arguments so that changing the argument doesn't mysteriously break this integration test.

Reply:
done